contributing-guides/git-terminal.md

```diff
@@ -29,3 +31,32 @@ Typically a pull request will include changes in a single file.
 [pr-howto]: ../CONTRIBUTING.md#submitting-a-pull-request
 [commit-msg]: ../CONTRIBUTING.md#commit-message
 [mass-changes]: https://github.com/tldr-pages/tldr/pulls?&q=is:pr+is:merged+label:"mass+changes"
+
+# Updating your fork
+
+Forks of GitHub repositories aren't updated automatically. In order to keep your fork of the repository up-to-date with the latest changes and avoid merge conflicts, you should update it regularly.
+
+There are two ways to update your fork.
+
+1. Via the GitHub web interface.  Click `Fetch upstream` and then `Fetch and merge` on the fork as shown below:
+
+![Fetch and merge button in GitHub](../images/github-fetch-and-merge-button.png).
+
+2. Using Git in the terminal:
+
+```bash
+git checkout main
+git remote add upstream https://github.com/tldr-pages/tldr.git  # only run if you don't already have the upstream remote (check with "git remote -v")
+git fetch upstream main
+git rebase upstream/main
```

Comment:
It might be worth linking to a resource, or adding a comment for what a user should do if they have merge conflicts?

Reply:
See also `git mergetool`; [Meld](https://meldmerge.org/)

Reply:
> It might be worth linking to a resource, or adding a comment for what a user should do if they have merge conflicts?

Sounds like a good idea. Could you suggest something? You prabably have far more experience with git than me.

Reply:
I wouldn't make that assumption!

I assume GitHub docs might be the best way to go?
https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

Reply:
Managing merge conficts in the command line isn't the easiest thing to do (especially for beginners), but sounds like the only choice we have. 

I'll add that link.

Reply:
Done (90336a3). I'm not sure about the wording though.